# Manipulation plan solution response
moveit_task_constructor_msgs/Solution solution

# Reason for error
moveit_msgs/MoveItErrorCodes error_code
